Document/0x06a-Platform-Overview.md

```diff
@@ -48,35 +48,33 @@ Signing application code in iOS is different than in Android. In the latter you
 At least two scenarios exists where you can install an application without the App Store:
 
 1. Via Enterprise Mobile Device Management. This requires the company to have company-wide certificate signed by Apple.
-2. Via side-loading - i.e. by signing the app with a developer's certificate and installing it on the device via Xcode. There is an upper limit of number of devices that can be used with the same certificate.
+2. Via side-loading, i.e. by signing an app with the developer's certificate and installing it on the device via Xcode. There is a top limit of the number of devices that can be used with the same certificate.
 
-A developer profile and a Apple-signed certificate is required in order to deploy and run an application.
-Developers need to register with Apple and join the [Apple Developer Program](https://developer.apple.com/support/compare-memberships/ "Membership for Apple Developer Program") and pay a yearly subscription fee to get the full range of development and deployment possibilities. A free account still allows you to compile and deploy an application via side-loading.  
+Developer profile and Apple-signed certificate is required in order to deploy and run an application.
+Developers need to register with Apple and join the [Apple Developer Program](https://developer.apple.com/support/compare-memberships/ "Membership for Apple Developer Program") paying a yearly subscription fee to get the full range of development and deployment possibilities. A free account still allows you to compile and deploy an application via side-loading.  
 
-Apple has implemented an intricate DRM system to make sure that only valid & approved code runs on Apple devices. In other words, on a non-jailbroken device, you won't be able to run any code unless Apple explicitly allows you to. You can't even opt to run code on your own device unless you enroll with the Apple developer program and obtain a provisioning profile and signing certificate. For this and other reasons, iOS has been [compared to a crystal prison](https://www.eff.org/deeplinks/2012/05/apples-crystal-prison-and-future-open-platforms "Apple's Crystal Prison and the Future of Open Platforms")
+Apple has implemented an intricate DRM system to make sure that only valid and approved code runs on Apple devices. In other words, on a non-jailbroken device, one will not be able to run any code unless Apple explicitly allows it. You cannot even opt to run any code on your own device unless you enrol to the Apple developer program and obtain the provisioning profile and signing certificate. For this and other reasons, iOS has been [compared to a crystal prison](https://www.eff.org/deeplinks/2012/05/apples-crystal-prison-and-future-open-platforms "Apple's Crystal Prison and the Future of Open Platforms").

```

Comment:
> unless you enrol to the Apple developer program

In American English, it is "enroll". I think you also enroll *in* a program, not *to* a program.